### PR TITLE
Minor terminology change

### PR DESCRIPTION
**Pull request by athousandwords**
These docs are great!! Just tweaking working (restart instead of reboot).

### CODE REVIEW

File: commandline/WSL/install_guide.md [hunk 1]
```diff
@@ -51,9 +51,9 @@ Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-L
 ```
 
 ### After enabling Windows Subsystem for Linux
-**Reboot when prompted**
+**Restart when prompted**
```
Comment (bitcrazed) [inline commandline/WSL/install_guide.md]: Hey Susan, thanks for your suggested edits - could I ask why the preference for restart over reboot? We tend to refer to "restarting" Bash by closing and re-opening the Bash console window. But when we say reboot, we mean rebooting your whole PC.

Comment (athousandwords) [inline commandline/WSL/install_guide.md]: It's just a Microsoft Style thing. The style guide is here, and you can search for "reboot" for the exact guidance. https://worldready.cloudapp.net/Styleguide/Read?id=2700

**Comment by bitcrazed** [comment]
Thanks for your edits Susan - great stuff.

LGTM :shipit: @jackchammons 😀

**Comment by jackchammons** [comment]
Thanks @athousandwords! Merging

**Pull request merged**